census

```diff
@@ -0,0 +1,621 @@
+#!/usr/bin/python
```

Comment:
done but we can ignore these bits until we decide what to do about the canonical-poke approach